ENIGMAsystem/SHELL/Widget_Systems/Unix/Info/About.ey

```diff
@@ -0,0 +1,9 @@
+%e-yaml
+---
+
+Name: Unix
+Identifier: Unix
+Build-Platforms: MacOSX, Linux
+Represents: MacOSX, Linux
```

Comment:
Represents makes it a default for said platform as it adds deps should represent none